### ISSUE TITLE
Multiple ^B

### ISSUE THREAD

**Issue by kensanata** [aaron-em/rcirc-styles.el#17]
Using the `mastodon-bitlbee` plugin, I get a bunch of messages like these when I log in:

> ^B[^B^B^B2018-02-12 22:40:20^B]^B ^B[^B00^B]^B bla bla bla

The parser will sometimes incorrectly leave a single ^B standing and leave the entire message ("bla bla bla") in bold. My IRC buffer then looks like this:

> **[**^B**2018-02-12 22:40:20**] [**00**] **bla bla bla**

**Comment by aaron-em** [comment]
Interesting! I use neither of those things, so am unable to reproduce the issue on my own: if you'll provide a repro that's reliable enough to use in a unit test, I'll find some time soon to correct this. Thanks for reporting it!

--
Aaron Miller
Mobile: (443) 902-4702
me@aaron-miller.me
https://aaron-m.com

**Comment by kensanata** [comment]
Gaaah, sorry for the noise. When I started looking for the problem, I could reproduce it until I started debugging it, at which point it was suddenly solved. Then I tried `emacs -q` and loaded the library manually and couldn't reproduce it. Finally I thought of using `list-load-path-shadows` and found a lonely `rcirc-styles.elc` (old, compiled, aaargh!) -- such that whenever I looked at the sources, it looked good, whenever I debugged the code, it was fixed. :)
